packages/intl/src/dateTime.ts

```diff
@@ -125,6 +125,8 @@ export function formatDateTimeRange(
   ...[from, to, options = {}]: Parameters<IntlFormatters['formatDateTimeRange']>
 ): string {
   const {timeZone, locale, onError} = config
+  const fromDate = typeof from === 'string' ? new Date(from || 0) : from
```

Comment:
I don't think you need this `typeof`

Reply:
Thanks for reviewing.

I was just following the same pattern as in `formatDate` for consistency: https://github.com/formatjs/formatjs/blob/main/packages/intl/src/dateTime.ts#L83

I can change it on both places to something else or keep it like this. Any preference?